server/app/views/applicant/NorthStarApplicantProgramBlockEditView.java

```diff
@@ -198,4 +198,24 @@ private void addFileUploadParameters(
                 .review(params.profile(), params.applicantId(), params.programId())
                 .url());
   }
+
+  @AutoValue
+  public abstract static class Params {
```

Comment:
Question: why create this additional class instead of adding programDescription to ApplicationBaseViewParams? It already has similar fields like programTitle.

Reply:
Good point